pkg/api/v1/types.go

```diff
@@ -1328,6 +1337,8 @@ type NodeSystemInfo struct {
 	ContainerRuntimeVersion string `json:"containerRuntimeVersion" description:"Container runtime version reported by the node through runtime remote API (e.g. docker://1.5.0)"`
 	// Kubelet version reported by the node
 	KubeletVersion string `json:"kubeletVersion" description:"Kubelet version reported by the node"`
+	// Open ports on Kubelet
+	KubeletPorts KubeletPorts `json:"kubeletPorts" description:"Information about ports open by Kubelet"`
```

Comment:
I don't agree that it should be put into Addresses field. The types don't match - Addresses stores IPs, while Ports stores ports. I think mixing those will be confusing. I can put there whole address "IP:KubeletPort" and call it KubeletAddress or something, but this will introduce redundancy with the Nodes address.

I think that SystemInfo is a good place for information about ports used by system daemons, but I'm completely OK with making it a top-level NodeDaemonPorts (or sth) field.

I'm not certain if those ports should be explicit, as in current proposal, or it should be put into the (logical) 'map', as NodeAddresses currently are. Having it explicit makes code simpler, as I don't need to do lookups in the slice (I honestly hate this pattern), but it makes API harder to evolve (which is why I'm not against it).